pkg/storage/SQL/plan_document_read_by_solution_id_not_restricted.sql

```diff
@@ -0,0 +1,23 @@
+SELECT
+    doc.id,
+    doc.model_plan_id,
+    doc.file_type,
+    doc.bucket,
+    doc.file_key,
+    doc.virus_scanned,
+    doc.virus_clean,
+    doc.file_name,
+    doc.file_size,
+    doc.restricted,
+    doc.document_type,
+    doc.other_type,
+    doc.optional_notes,
+    doc.deleted_at,
+    doc.created_by,
+    doc.created_dts,
+    doc.modified_by,
+    doc.modified_dts
+FROM plan_document AS doc
+INNER JOIN plan_document_solution_link AS link ON link.document_id = doc.id
+INNER JOIN operational_solution AS sol ON sol.id = link.id
```

Comment:
I think this is what breaks the resolver. I think it should be handled like it is in the other  SQL file. I tested it by querying from my local SQL tool. 
```suggestion
FROM plan_document AS doc
INNER JOIN plan_document_solution_link AS link ON link.document_id = doc.id
WHERE link.solution_id = :solution_id AND doc.restricted = FALSE
```

Reply:
Yes, I had a batch of changes to push up